x-pack/plugins/index_management/public/application/components/component_templates/lib/api.ts

```diff
@@ -97,7 +97,7 @@ export const getApi = (
 
   async function postDataStreamRollover(name: string) {
     return sendRequest<ComponentTemplateDatastreams>({
-      path: `${apiBasePath}/data_streams/${encodeURIComponent(name)}/rollover`,
+      path: `${apiBasePath}/data_streams/${encodeURIComponent(name)}/rollover?lazy`,
```

Comment:
we should revert this for now because of https://github.com/elastic/kibana/pull/175504

Reply:
Yes I will remove it